packages/editor/src/browser/editor-keybinding.ts

```diff
@@ -31,21 +30,6 @@ export class EditorKeybindingContribution implements KeybindingContribution {
     ) { }
 
     registerKeyBindings(registry: KeybindingRegistry): void {
-        [
-            {
-                commandId: 'editor.close',
-                context: this.editorKeybindingContext,
-                keyCode: KeyCode.createKeyCode({ first: Key.KEY_W, modifiers: [Modifier.M3] })
-            },
-            {
-                commandId: 'editor.close.all',
-                context: this.editorKeybindingContext,
-                keyCode: KeyCode.createKeyCode({ first: Key.KEY_W, modifiers: [Modifier.M2, Modifier.M3] })
-            }
-        ].forEach(binding => {
-            registry.registerKeyBinding(binding);
-        });
-
```

Comment:
Is there a good reason to keep this?

Reply:
just don't want to remove today to add it tomorrow